src/extension.ts

```
@@ -79,10 +79,11 @@ async function getTargetServer(tomcat: TomcatController, tomcatItem ?: TomcatSer
     }
 
     if (!server) {
-        return Promise.reject(new Error(Utility.localize('tomcatExt.noserver', 'Tomcat server is undefined')));
-    } else {
-        return server;
+        vscode.window.showInformationMessage(Utility.localize('tomcatExt.noservertodelete', 'No tomcat server.'));
+        return null;
```

return null will make the command fail silently.